hot-reload-analysis/src/main/kotlin/org/jetbrains/compose/reload/analysis/InstructionToken.kt

```diff
@@ -283,18 +287,12 @@ private object EndRestartGroupTokenizer : InstructionTokenizer() {
 
 private object StartReplaceGroupTokenizer : InstructionTokenizer() {
     override fun nextToken(context: TokenizerContext): Either<InstructionToken, Failure>? {
-        val expectedLdc = context[0] ?: return null
-        val expectedMethodInsn = context[1] ?: return null
-
-        if (expectedMethodInsn is MethodInsnNode &&
-            MethodId(expectedMethodInsn) == Ids.Composer.startReplaceGroup
-        ) {
-            val keyValue = expectedLdc.intValueOrNull() ?: return Failure(
-                "Failed parsing startReplaceGroup token: expected key value"
-            ).toRight()
-
-            return StartReplaceGroup(ComposeGroupKey(keyValue), listOf(expectedLdc, expectedMethodInsn)).toLeft()
+        val consumer = context.consumer()
```

Comment:
`SourceInformationTokenizer::nextToken` also expects LDC + MethodInsn, maybe we can update it to use `findMethodInsn` as well?

Reply:
Same with `EndToMarkerTokenizer`, it expects ILOAD + MethodInsn